.github/workflows/coverage.yml

```
@@ -0,0 +1,86 @@
+      - name: Run Tests with Coverage
+        run: dotnet test --configuration Release --no-build --collect:"XPlat Code Coverage" --logger "trx;LogFileName=test_results.trx" --results-directory ./TestResults
+
+      - name: Check Coverage Threshold
+        run: |
+          # Parse coverage XML and check if line coverage >= 90%
+          python3 -c "
+          import xml.etree.ElementTree as ET
+          import sys
+          import glob
+          import os
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)